### PR TITLE
Team

### PR DESCRIPTION
**Pull request by g4s8**
#527 - renamed gang to team

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#711](https://github.com/zerocracy/farm/issues/711) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#711](https://github.com/zerocracy/farm/issues/711) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/620859651). The budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4). Please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

**Comment by carlosmiranda** [comment]
@yegor256 this is good to merge

File: src/main/java/com/zerocracy/tk/TkApp.java [hunk 1]
```diff
@@ -198,7 +198,7 @@ public TkApp(final Farm farm, final Props props,
                                                                 new FkRegex("/rfp-charge", new TkPrepay(farm)),
                                                                 new FkRegex("/rfp-post", new TkSubmit(farm)),
                                                                 new FkRegex("/board", new TkBoard(farm)),
-                                                                new FkRegex("/gang", new TkGang(farm)),
```
Comment (yegor256) [inline src/main/java/com/zerocracy/tk/TkApp.java]: @g4s8 we should forward `/gang` to `/team`, since we have it in may places already

**Comment by yegor256** [comment]
@g4s8 see above

**Comment by g4s8** [comment]
@yegor256 fixed, see new commit

**Comment by 0crat** [comment]
Order was finished, quality was "acceptable": +15 points just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
The job [#711](https://github.com/zerocracy/farm/issues/711) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Pull request merged**